src/scss/components/_navbar.scss

```diff
@@ -184,6 +184,10 @@
     }
     .Navbar-link {
       color: $base-grey;
+
+      &:hover {
+        color: #000;
```

Comment:
This change is too subtle for anyone to notice. If we can't use the interactive color of each page, let's just use the underline decoration.